lib/yaml-lint.js

```diff
@@ -13,11 +13,7 @@ function gulpYamlLint (options = {}) {
     transform (file, enc, cb) {
       let errCount = 0;
```

Comment:
this could be also a boolean flag according to the current logic

Reply:
`message: `Failed with ${errCount} ${errCount === 1 ? 'error' : 'errors'}`,` wouldn't work then.